lessonbuilder/tool/src/webapp/js/show-page.js

```diff
@@ -2977,6 +3001,24 @@ $(function () {
   });
 
   function mmFileInputDelete() {
+    let wasOverUploadSize = accumulatedFileSize / 1024 / 1024 > maxFileUploadSize ;
```

Comment:
Use 1024 as a const

Reply:
Hi - I have added 'const ONE_KB = 1024' and used it in place of 1024 literals within show-page.js
Thanks for the suggestion.